topics/sequence-analysis/tutorials/ncbi-fcs/tutorial.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: tutorial_hands_on
+
+title: Screening assembled genomes for contamination using NCBI FCS
+zenodo_link: 'https://zenodo.org/records/10881551'
+questions:
+- Are the sequences in a genome assembly contaminant-free?
+objectives:
+- Learn how to screen a genome assembly for adaptor and vector contamination.
+- Learn how to screen a genome assembly for non-host organism contamination.
+time_estimation: 90M
+key_points:
+- Contamination can arise from multiple sources and occur at multiple stages of a genome assembly project.
+- Contamination cleanup is essential for reliable comparative genomics analyses.
+contributors:
+- tvedtees
+
+---
+
+
+# Introduction
+
+<!-- This is a comment. -->
+
+The National Center for Biotechnology Information (NCBI) performs contamination screening of genome assemblies submitted to the archival GenBank. Advances in genome sequencing have accelerated the production of genome assemblies and their submission to public databases, necessitating high-performance screening tools. Since contaminants can lead to misleading conclusions about the biology of the organism in question (e.g. gene content, gene evolution, etc.), ideally contamination screening should be performed after the initial contig assembly and prior to downstream genome analyses. 
+
+NCBI has released a publicly-available Foreign Contamination Screen (FCS) tool suite to detect contaminants from various sources and produce a cleaned sequence set. This tutorial provides a quick example of two current FCS tools: FCS-adaptor identifies synthetic sequences used in library preparation, and FCS-GX {% cite Astashya2024 %} identifies sequences from foreign organisms assigned to discordant taxonomies compared to the user-declared source organism.
```

Comment:
If you have a lot of abbreviations used throughout that you expect readers will not be familiar with you might be intersted in using the [GTN's abbreviation feature](https://training.galaxyproject.org/training-material/news/2021/05/25/abbreviations-tag.html) (completely optional!)

Reply:
Thank you for the tip. NCBI is familiar to users, and FCS is the only other commonly used abbreviation.